system/dtb/README.md

```diff
@@ -0,0 +1,17 @@
+<!--
+SPDX-FileCopyrightText: 2021 Armijn Hemel <armijn+github@tjaldur.nl>
+
+SPDX-License-Identifier: CC0-1.0
+-->
+
+## xenvm-4.2.dts
+
+Source: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/arch/arm/boot/dts/xenvm-4.2.dts?id=b244131
```

Comment:
A commit hash shortened to 7 characters is fine for most repos (that usually have only a few hundred commits), but the [Linux kernel](https://github.com/torvalds/linux) has over a _million_ commits, so there is a non-negligible chance that the 7-character abbreviation you pick will become ambiguous as new commits are added.

An example of an ambiguous 7-character short SHA-1 hash is mentioned in [this post](https://github.blog/2016-11-29-git-2-11-has-been-released/#abbreviated-sha-1-names) - this commit https://github.com/torvalds/linux/commit/6d7942dc2a70a7e74c352107b150265602671588 references another commit via a short SHA-1 name `78a8b35` in the commit message, but <a href="https://github.com/torvalds/linux/commit/78a8b35">https:// github.com/torvalds/linux/commit/78a8b35</a> will yield 404 Not Found because there are multiple commits that this reference might lead to.

So for large repos like this, it's recommended to include a bit longer hash - 12 characters should be enough.